src/solver/type_getters.jl

```diff
@@ -479,19 +479,19 @@ function get_sim_info(config::AtmosConfig)
     (; parsed_args) = config
     FT = eltype(config)
 
-    job_id = if isnothing(parsed_args["job_id"])
-        job_id_from_config(parsed_args)
-    else
-        parsed_args["job_id"]
-    end
+    (; job_id) = config
     default_output = haskey(ENV, "CI") ? job_id : joinpath("output", job_id)
     out_dir = parsed_args["output_dir"]
     base_output_dir = isnothing(out_dir) ? default_output : out_dir
 
-    output_dir = OutputPathGenerator.generate_output_path(
-        base_output_dir;
-        context = config.comms_ctx,
-    )
```

Comment:
@charleskawczynski Any reason this is removed in this PR?

Reply:
I removed it because of of https://github.com/CliMA/ClimaUtilities.jl/issues/56. `generate_output_path` was breaking with libuv errors: https://github.com/CliMA/ClimaAtmos.jl/actions/runs/9020764355/job/24786627579?pr=2994#step:6:1489.

I think that code needs to be made more robust, and there have been other issues, preventing me from running simulations on windows (e.g., https://github.com/CliMA/ClimaUtilities.jl/issues/52).

Reply:
Ah, I see, thanks!